sprinter/formula/eggscript.py

```diff
@@ -74,7 +74,7 @@ def __install_eggs(self, config):
                   'w+') as fh:
             fh.write('\n'.join(eggs))
         stdout = subprocess.PIPE if config.is_affirmative('redirect_stdout_to_log', 'true') else None
-        return_code, output = lib.call("bin/pip install -r requirements.txt --upgrade",
+        return_code, output = lib.call("PYTHONPATH="" bin/pip install -r requirements.txt --upgrade",
```

Comment:
I'm not sure if this has the intended effect. you have two double quotes nested inside double quotes, so this will result in a concatenation of two strings in Python.

You can use single quotes to get this effect instead.